src/common/InfoLabel.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
We have an OfflineNotice component that does a very similar thing. I guess we either want to merge this one with it, or make this a more specific one like 'UnreadMsgNotice'.